### ISSUE TITLE
Fresh RISE install in new conda environment does not work

### ISSUE THREAD

**Issue by ThomasA** [damianavila/RISE#369]
I have used RISE for a long time. Now I wanted to enable it in a new conda environment where I had not installed it previously and it did not seem to be available from the global anaconda environment.
In the environment in question, I ran `conda install -c damianavila82 rise` and the install completed without errors. When I start `jupyter notebook` afterwards and create Python notebook, the usual slideshow button for RISE is missing:
![screenshot-2018-4-19 l07_gpucomp](https://user-images.githubusercontent.com/971307/38985925-13949fc4-43cb-11e8-91e8-fdb45e7971d8.png)

The environment is Anaconda 5.0.1 running Python 3.6, conda is version 4.3.27, and Jupyter is version 4.3.0.
How can can I figure out what is going on here?

**Comment by damianavila** [comment]
That's correct... there seems to be some issue with some versions of the notebook in the 5.x series... please try to upgrade the notebook to the latest version and see if that fix the issue.

**Comment by damianavila** [comment]
Btw, to check the notebook version in conda you can do a `conda list notebook`.

**Comment by ThomasA** [comment]
I am not quite sure how the different components of Jupyter map to Anaconda packages. I currently have:

    jupyter                   1.0.0            py36h9896ce5_0  
    jupyter_client            5.1.0            py36h614e9ea_0  
    jupyter_console           5.2.0            py36he59e554_1
    jupyter_core              4.3.0            py36h357a921_0  
    jupyterlab                0.27.0           py36h86377d0_2  
    jupyterlab_launcher       0.4.0            py36h4d8058d_0  
    notebook                  5.0.0            py36h0b20546_2

I wonder what the difference is between the `jupyter*` packages and the `notebook` package?
I guess I need to upgrade `jupyter_core` to 4.4.0 and `notebook` to 5.4.0 here for testing?

**Comment by damianavila** [comment]
You just need to upgrade the `notebook` to the latest available with `conda update notebook`. That should be sufficient.

**Comment by ThomasA** [comment]
I just installed a new conda environment with Anaconda 5.1.0. That comes with the mentioned version of Jupyter and seems to have solved the problem. Thanks for the help, but I am little surprised to see that backward compatibility is apparently this short-lived.

**Comment by damianavila** [comment]
> Thanks for the help, but I am little surprised to see that backward compatibility is apparently this short-lived.

In fact, we have published versions for several of the notebook version series: https://damianavila.github.io/RISE/changelog.html
I think the problem here is specifically with some of the 5.X versions... an upstream problem, in the notebook itself affecting some of the 5.X versions.

**Comment by parmentelat** [comment]
Please bear in mind that the jupyter ecosystem is moving full speed; in this context, version 5.0.0 of the notebook, that was issued a little more than one year ago, feels a bit like prehistory ;)

If you're curious to see all that's happened since then:
http://jupyter-notebook.readthedocs.io/en/stable/changelog.html

**Comment by Saho12** [comment]
Can anyone please help me on this 
https://github.com/damianavila/RISE/issues/420#issuecomment-626047346
